### PR TITLE
FACES-3236 Extract Liferay specific TCK tester code into separate jar

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: tck/bridge-tck-main-portlet/pom.xml [hunk 1]
```diff
@@ -85,6 +85,7 @@
 		<portlet.container>liferay</portlet.container>
 		<integration.container>${portlet.container}</integration.container>
 		<portal.version.major>${liferay.version.major}</portal.version.major>
+		<tck.test.case.class>LiferayTckTestCase</tck.test.case.class>
```
Comment (ngriffin7a) [inline tck/bridge-tck-main-portlet/pom.xml:88]: @stiemannkj1 This assumes Liferay is the default, and uses the "pluto" profile to override. I think we normally default to Pluto, and then override with Liferay specific stuff in the "liferay" profile. No?

**Comment by ngriffin7a** [comment]
bp 5.x 4.x 3.x 2.x <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ngriffin7a** [comment]
Merged, thank you. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: tck/bridge-tck-main-portlet/pom.xml (hunk 1 above)
Comment (stiemannkj1) [reply tck/bridge-tck-main-portlet/pom.xml:88]: @ngriffin7a, we default to testing in Liferay to be consistent with the integration tests.

**Pull request merged**